packages/lz4_chans/lz4_chans.2.0.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/lz4-chans"
+bug-reports: "https://github.com/UnixJunkie/lz4-chans/issues"
+dev-repo: "git+https://github.com/UnixJunkie/lz4-chans.git"
+license: "BSD-3"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "base-unix"
+  "dolog"
+  "dune" {build}
+]
+depexts: [
```

Comment:
for future releases you might want to create a separate conf package for this. It's easier to maintain.